starters/graphql-starter/.env.example

```diff
@@ -1,10 +1,15 @@
 # See https://next-drupal.org/docs/environment-variables
```

Comment:
The changes to `graphql-starter` are almost identical to the changes in the `basic-starter`.